### PR TITLE
Road Property Set fixes

### PR DESCRIPTION
**Pull request by AlexBrad1eyCT**
P&W Property set fixes highlighted during deliverable import.

Fixes Road responsibilities of #326

### CODE REVIEW

**Comment by AlexBrad1eyCT** [comment]
@larswik a check through would be appreciated as these are road fixes!

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_CourseApplicationConditions/DocPropertySet.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CourseApplicationConditions" UniqueId="1b6cb3aa-95e9-4fec-8d1a-d6d502edb484" ApplicableType="IfcCourse" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CourseApplicationConditions" UniqueId="1b6cb3aa-95e9-4fec-8d1a-d6d502edb484" ApplicableType="IfcCourse,IfcCourseType" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (larswik) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_CourseApplicationConditions/DocPropertySet.xml]: I am wondering if this should be PSET_OCCURRENCEDRIVEN applicable to IfcCourse only?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_MarkingLinesCommon/DocPropertySet.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MarkingLinesCommon" UniqueId="a292e50a-6238-42bf-8cbf-70af606b496c" ApplicableType="IfcSurfaceFeature/(.LINEMARKING.)">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MarkingLinesCommon" UniqueId="a292e50a-6238-42bf-8cbf-70af606b496c" ApplicableType="IfcSurfaceFeature/LINEMARKING" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (larswik) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_MarkingLinesCommon/DocPropertySet.xml]: PSET_OCCURRENCEDRIVEN?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_PlantCommon/DocPropertySet.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_PlantCommon" UniqueId="bff7eedc-8844-446a-b6c2-fd5bdaad0968" ApplicableType="IfcPlant">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_PlantCommon" UniqueId="bff7eedc-8844-446a-b6c2-fd5bdaad0968" ApplicableType="IfcPlant" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (larswik) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_PlantCommon/DocPropertySet.xml]: PSET_OCCURRENCEDRIVEN?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_RoadMarkingCommon/DocPropertySet.xml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RoadMarkingCommon" UniqueId="e10080b5-0bcc-453a-b230-5c72efe09fb4" ApplicableType="IfcSurfaceFeature/(.HATCHMARKING.),IfcSurfaceFeature/(.LINEMARKING.),IfcSurfaceFeature/(.PAVEMENTSURFACEMARKING.),IfcSurfaceFeature/(.SYMBOLMARKING.)">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RoadMarkingCommon" UniqueId="e10080b5-0bcc-453a-b230-5c72efe09fb4" ApplicableType="IfcSurfaceFeature/HATCHMARKING,IfcSurfaceFeature/LINEMARKING,IfcSurfaceFeature/PAVEMENTSURFACEMARKING,IfcSurfaceFeature/SYMBOLMARKING" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (larswik) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_RoadMarkingCommon/DocPropertySet.xml]: PSET_OCCURRENCEDRIVEN?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_RoadSymbolsCommon/DocPropertySet.xml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RoadSymbolsCommon" UniqueId="1beaa2e6-a4a1-4d14-a2c7-ba651e6baec6" ApplicableType="IfcSurfaceFeature/(.SYMBOLMARKING.)">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RoadSymbolsCommon" UniqueId="1beaa2e6-a4a1-4d14-a2c7-ba651e6baec6" ApplicableType="IfcSurfaceFeature/SYMBOLMARKING" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (larswik) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_RoadSymbolsCommon/DocPropertySet.xml]: PSET_OCCURRENCEDRIVEN?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_CourseApplicationConditions/DocPropertySet.xml (hunk 1 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_CourseApplicationConditions/DocPropertySet.xml]: Yes this looks occurence driven will update

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_RoadMarkingCommon/DocPropertySet.xml (hunk 4 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_RoadMarkingCommon/DocPropertySet.xml]: yes will change.

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_RoadSymbolsCommon/DocPropertySet.xml (hunk 5 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_RoadSymbolsCommon/DocPropertySet.xml]: yes also

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_PlantCommon/DocPropertySet.xml (hunk 3 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_PlantCommon/DocPropertySet.xml]: uumm not sure, i know it doesnt have a type yet but..

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_MarkingLinesCommon/DocPropertySet.xml (hunk 2 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_MarkingLinesCommon/DocPropertySet.xml]: yep will change

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_BoundedCourseCommon/DocPropertySet.xml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_BoundedCourseCommon" UniqueId="75246832-754b-4570-a4e2-24baa7a30a33" ApplicableType="IfcCourse" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_BoundedCourseCommon" UniqueId="75246832-754b-4570-a4e2-24baa7a30a33" ApplicableType="IfcCourse,IfcCourseType" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_BoundedCourseCommon/DocPropertySet.xml]: I am a bit skeptical about `IfcObjectType` applicabilities. These were supposed to be handled on occurrence level. If this is supposed to change, sure but we need a common agreement on this.

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_RoadMarkingCommon/DocPropertySet.xml [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RoadMarkingCommon" UniqueId="e10080b5-0bcc-453a-b230-5c72efe09fb4" ApplicableType="IfcSurfaceFeature/(.HATCHMARKING.),IfcSurfaceFeature/(.LINEMARKING.),IfcSurfaceFeature/(.PAVEMENTSURFACEMARKING.),IfcSurfaceFeature/(.SYMBOLMARKING.)">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RoadMarkingCommon" UniqueId="e10080b5-0bcc-453a-b230-5c72efe09fb4" ApplicableType="IfcSurfaceFeature/HATCHMARKING,IfcSurfaceFeature/LINEMARKING,IfcSurfaceFeature/PAVEMENTSURFACEMARKING,IfcSurfaceFeature/SYMBOLMARKING" PropertySetType="PSET_OCCURRENCEDRIVENa">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_RoadMarkingCommon/DocPropertySet.xml]:
```suggestion
<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RoadMarkingCommon" UniqueId="e10080b5-0bcc-453a-b230-5c72efe09fb4" ApplicableType="IfcSurfaceFeature/HATCHMARKING,IfcSurfaceFeature/LINEMARKING,IfcSurfaceFeature/PAVEMENTSURFACEMARKING,IfcSurfaceFeature/SYMBOLMARKING" PropertySetType="PSET_OCCURRENCEDRIVEN">
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_PlantCommon/DocPropertySet.xml (hunk 3 above)
Comment (SergejMuhic) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_PlantCommon/DocPropertySet.xml]: Should this not be the same as with other entities that do not have a type?

**Review by SergejMuhic — CHANGES_REQUESTED** [review]

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_BoundedCourseCommon/DocPropertySet.xml (hunk 6 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_BoundedCourseCommon/DocPropertySet.xml]: @SergejMuhic @larswik needs resolving

**Comment by AlexBrad1eyCT** [comment]
there are a few errors corrected in this and again the Type issue

**Review by SergejMuhic — APPROVED** [review]

**Comment by SergejMuhic** [comment]
@larswik I will merge this. Feel free to look at it later. We can still change.

**Pull request merged**